installer/dev/InstallActivityContext.cpp

```diff
@@ -65,7 +65,7 @@ BOOL WindowsAppRuntimeInstaller::InstallActivity::Context::LogInstallerCommandLi
         WCHAR message[1024]{};
         PCWSTR messageFormat{ L"Windows App Runtime Installer (ActivityId: %s) is called with Command Line Args: %s" };
         auto installerActivityId{ winrt::to_hstring(*m_activity.Id()) };
-        FAIL_FAST_IF_FAILED(StringCchPrintfW(message, ARRAYSIZE(message), messageFormat, installerActivityId, cmdlineArgs));
+        FAIL_FAST_IF_FAILED(StringCchPrintfW(message, ARRAYSIZE(message), messageFormat, installerActivityId.data(), cmdlineArgs));
```

Comment:
@bpulliam had concern about whether winrt::data() was safe for empty strings.  appears so:
https://github.com/microsoft/cppwinrt/blob/9b453cfc518bdaa7e2ff590526c4883457fd6065/strings/base_string.h#L272C28-L272C28